server/api/nyt.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
```

Comment:
@AnnaC91 these models need to be updated, it shouldn't take like select text cmd+d select all and change every instance of Nyt to article